runners/gradle-plugin/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ import org.jetbrains.*
 
 plugins {
     `java-gradle-plugin`
```

Comment:
You could remove this too, the `kotlin-dsl` applies the `JavaGradlePluginPlugin`.

Reply:
I'll merge this PR now to get the whole chain moving, but @aSemy if you want to apply the suggestion, I think we can do it in #2833, since it'll have to be rebased anyway